app/src/main/res/xml-eu/ext_kbd_bottom_row_ng.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm not sure this is the best way to do this. I sometimes change those bottom rows, and I plan to add another bottom row. I most likely forget to change the Basque-specific rows.